package.json

```diff
@@ -11,7 +11,8 @@
   },
   "dependencies": {
     "react": "17.0.2",
-    "react-native": "0.67.3"
+    "react-native": "0.67.3",
+    "react-native-vector-icons": "^9.1.0"
```

Comment:
Note: if you're using this component, you need to run `npx react-native link` to to let `react-native-vector-icons` library render icons properly.